includes/Traits/Publisher.php

```diff
@@ -125,26 +125,8 @@ public function get_publisher_logo() {
 			$logo_image_url = $this->get_valid_publisher_image( $active_publisher_logo );
 		}
 
-		// @todo Once we are enforcing setting publisher logo in the editor, we shouldn't need the fallback options.
-		// Currently, it's marked as required but that's not actually enforced.
-
-		// Finding fallback image.
-		$custom_logo_id = get_theme_mod( 'custom_logo' );
-		if ( empty( $logo_image_url ) && has_custom_logo() && $custom_logo_id ) {
-			$logo_image_url = $this->get_valid_publisher_image( $custom_logo_id );
-		}
-
-		// Try Site Icon, though it is not ideal for non-Story because it should be square.
-		$site_icon_id = get_option( 'site_icon' );
-		if ( empty( $logo_image_url ) && $site_icon_id ) {
-			$logo_image_url = $this->get_valid_publisher_image( $site_icon_id );
-		}
-
 		// Fallback to serving the WordPress logo.
 		$placeholder = $this->get_publisher_logo_placeholder();
```

Comment:
The WordPress logo placeholder needs to be removed

Reply:
That also means removing the `fallback-wordpress-publisher-logo.png` file

Reply:
It is needed for the filter, 
https://github.com/google/web-stories-wp/blob/f2c821751fbe57f0894bc7af7e8bc68b5ba2ed82/includes/Traits/Publisher.php#L159

I also think there is a possibility that stories might be hotlinking to the image / url for the placeholder. Is it a problem if we remove it?

Reply:
Let's remove it

Reply:
I have marked this as deprecated but I did not remove placeholder, as it may break others plugins / code.